tests/platform_tests/api/test_psu.py

```diff
@@ -66,11 +65,10 @@ def compare_value_with_platform_facts(self, key, value, psu_idx):
             if expected_psus:
                 expected_value = expected_psus[psu_idx].get(key)
 
-        if not expected_value:
-            logger.warning("Unable to get expected value for '{}' from platform.json file for PSU {}".format(key, psu_idx))
-            return
+        pytest_assert(expected_value is not None,
```

Comment:
You should use `self.expect()` here, as this gets called on a number of devices in the list, this way we can gather all the error messages. Please refer ot my SFP changes in https://github.com/Azure/sonic-mgmt/pull/2268, not the chassis changes.